Monika After Story/game/script-topics.rpy

```diff
@@ -15694,6 +15696,134 @@ label monika_eating_meat:
+            m 5eubfb "I'm sure you are a really fun person to be around, [player].{w=0.3} I can't wait to be with you for real~"
+            m 5rkblsdra "Although, I won't hide the fact that I do enjoy the occasional moment of peace as well. {w=0.3}{nw}"
+            extend 5hksdrb "I hope you don't mind if I'm not always able to keep up, ahaha!"
+
+        "I'm somewhat in-between.":
+            $ persistent._mas_pm_ambivert = True
+            $ persistent._mas_pm_introvert = False
+            $ persistent._mas_pm_extrovert = False
+            $ persistent._mas_pm_social_personality_unsure = False
+            m 2hua "Ehehe, kind of like me, then."
+            m 2eud "Apparently, most people have both an introverted and extroverted side to their personality."
+            m 7eua "Even if usually, one of the two is dominant over the other, depending on the person."
+            m 7rsc "In our case, I guess not being too much on either side has both its positives and its negatives."
+            m 7rsd "Like, the good thing with people like us is that whe can feel comfortable both when being around people, and when spending some time alone."
+            m 7esc "And as for the negatives, I don't know if it's the same with you, but I've always had a hard time making deep, genuine connections with others..."
+            m 1eud "It's like, I have an easier time understanding most people, sure, but it doesn't mean I can always relate with them, you know ?"
+            m 1lksdld "So yeah, I end up being on good terms with almost everyone, but the friendships I form can sometimes feel a bit...{w=0.4}unfullfilling."
+            m 3eksdlc "The same thing happened with the club, for example."
+            m 3dksdld "I was so convinced that by bringing people together around something I truly enjoy, I'd have a better chance at bonding with them over our shared interests..."
+            show monika 5rkd at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5rkd "...and at the end of the day, we spent most of our time silently hanging out, everyone minding their own business."
+            m 5eka "Well, no point thinking about that."
+            m 5eubla "After all, I {i}did{/i} end up connecting in a meaningful way with a certain someone. {w=0.3}{nw}"
+            extend 5fubfu "A very charming someone, might I add~"
+
+        "I'm not really sure.":
+            $ persistent._mas_pm_social_personality_unsure = True
+            $ persistent._mas_pm_introvert = None
+            $ persistent._mas_pm_extrovert = None
+            $ persistent._mas_pm_ambivert = None
+            m 4ekb "That's alright, [player].{w=0.4} Things like that aren't always so clear."
+            m 4eua "I'm a little like you on that front."
+            m 2eka "I know I told you I was more on the extroversion side of things, but I still need a moment of tranquility to relax every once in a while, you know ?"
```

Comment:
Doesn't this go against what she mentions earlier? She says she's in-between, however now she's going back on herself on that.